client/.env

```diff
@@ -1,3 +1,4 @@
 VITE_MAPBOX_API_KEY=pk.eyJ1IjoiYmx1ZW9yYyIsImEiOiJja205bm12ajIxNGV0MndsYWVqeDc4enE3In0.38AVZ2_Gjy_ST_4LxcgahA
 VITE_PAGE_TITLE_PREFIX="XCCup.net - "
 VITE_BASE_TZ=Europe/Berlin
+VITE_BASE_USE_LIVE_API=false # Remember to allow CORS in browser;)
```

Comment:
Wofür?

Dafür haben wir doch verschiedene ENV-Files?
D.h. das müsste ich auch vor dem nächsten Deploy anpassen.

Reply:
Damit mann das Dev Frontend mit den Daten der Live API verwenden kann. Für den Deploy egal. In Produktion nimmt er immer den richtigen Endpunkt.

Reply:
Ah... clever ;-)